adoc/chapters/architecture.adoc

```diff
@@ -559,10 +559,10 @@ image::{images}/three-cg-three-queue.svg[align="center",opts="{imageopts}"]
 ==== Controlling execution order with events
 
 Submitting an action for execution returns an [code]#event# object. Programmers
-may use these events to explicitly synchronize programs. Host code can wait for an
-event to complete, which will block execution on the host until the action represented
-by the event has completed.  The [code]#event# class is described in greater detail
-in <<sec:interface.event>>.
+may use these events to explicitly coordinate host and device execution. Host
+code can wait for an event to complete, which will block execution on the host
+until the action(s) represented by the event has completed.  The [code]#event#
```

Comment:
```suggestion
until the action(s) represented by the event have completed.  The [code]#event#
```

I think we normal use plural grammar when using "(s)".

Reply:
Good catch.